### PR TITLE
Render post-workflow postables once workflow is complete

### PR DESCRIPTION
**Pull request by lukemelia**
- extract PostableCollection to support Milestone and use for Workflow#epilogue as well

CS-594

### CODE REVIEW

File: packages/web-client/tests/integration/components/workflow-thread-test.ts [hunk 1]
```diff
@@ -26,46 +28,74 @@ module('Integration | Component | workflow-thread', function (hooks) {
...
+      }),
+    ]);
+    this.set('workflow', workflow);
+    workflow.attachWorkflow();
+    await render(hbs`
+      <WorkflowThread @workflow={{this.workflow}} />
+    `);
+    assert.dom('[data-test-epilogue][data-test-postable]').doesNotExist();
+    postable1.isComplete = true;
+    await settled();
```
Comment (Aierie) [inline packages/web-client/tests/integration/components/workflow-thread-test.ts:96]: This is interesting. The tests fail if I remove this, but I can't see what asynchronous work is going on that would require settled - is it simply the DOM changes?

**Review by Aierie — APPROVED** [review]

File: packages/web-client/tests/integration/components/workflow-thread-test.ts (hunk 1 above)
Comment (lukemelia) [reply packages/web-client/tests/integration/components/workflow-thread-test.ts:96]: @Aierie `settled` waits for any "test waiters" to finish, including any queued tasks on the Ember run loop. An alternative that might be clearer would be to wait for the DOM state that we expect.

Comment (Aierie) [reply packages/web-client/tests/integration/components/workflow-thread-test.ts:96]: I see, thanks. So these waiting items might block ember from rerendering?

Comment (lukemelia) [reply packages/web-client/tests/integration/components/workflow-thread-test.ts:96]: No, the `settled` test waiter "waits" for Ember to complete rendering (among other things). There are a bunch of layers here, but you can have a look at https://github.com/emberjs/ember-test-helpers/blob/master/addon-test-support/%40ember/test-helpers/settled.ts as a starting point.

That being said, this conversation has convinced me I should use an approach that is more intention-revealing, so I'm going to make that change.

**Pull request merged**